packages/migrate/src/commands/DbPull.ts

```diff
@@ -468,29 +486,59 @@ ${`Run ${chalk.green(getCommandWithExecutor('prisma generate'))} to generate Pri
           warning.code === 31
         ) {
           message += warning.affected.map((it) => `- Model "${it.model}"`).join('\n')
-        } else if (warning.code === 20) {
+        } else if (
+          // * AffectedModelAndConstraint
+          warning.code === 35
+        ) {
+          message += warning.affected.map((it) => `-Model "${it.model}", Constraint "${it.constraint}"`).join('\n')
+        } else if (
+          // * AffectedTopLevel
+          warning.code === 20
```

Comment:
Could we use a numeric enum for these cases?

```ts
enum Response {
  No = 0,
  Yes = 1,
}
```

Reply:
What do you mean?

Reply:
Explanation over here https://prisma-company.slack.com/archives/C02FNFLDUS3/p1681740382535349?thread_ts=1681401438.067269&cid=C02FNFLDUS3

Reply:
Resolved per [slack thread](https://prisma-company.slack.com/archives/C02FNFLDUS3/p1681740524693499?thread_ts=1681401438.067269&cid=C02FNFLDUS3)